pages/common/ogrinfo.md

```diff
@@ -15,6 +15,18 @@
 
 `ogrinfo {{path/to/input.gpkg}} {{layer_name}}`
 
-- Only show summary information about a specific layer of a data source:
+- Show summary information about a specific layer of a data source:
 
 `ogrinfo -so {{path/to/input.gpkg}} {{layer_name}}`
+
+- Show summary of all layers of the data source:
+
+`ogrinfo -so -al {{path/to/input.gpkg}}`
+
+- Show detailed information of features matching a condition:
+
+`ogrinfo -where '{{myProperty > 42}}' {{path/to/input.gpkg}} {{layer_name}}`
```

Comment:
is this a property or an attribute?

Reply:
Good catch. I think both terms might be applicable, but `attribute` might be better. I will update it.

Reply:
I fixed it in [a009629](https://github.com/tldr-pages/tldr/pull/7904/commits/a009629240b89c276b82cfac12f81d32c2488cb2). 
I resolve the converstation. 
Please reopen if you want.